.github/workflows/CI.yml

```diff
@@ -14,6 +14,7 @@ jobs:
         with:
           node-version: '14'
       - run: npm ci
+      - run: npm run format.ci
```

Comment:
This script is missing from package.json. I think that's why the build is failing